contracts/Bondv4.sol

```diff
@@ -225,18 +224,24 @@ contract Bond is iBEP20 {
         }
         return true;
     }
+    function claimForMember(address asset) external returns (bool){
+        uint claimA = calcClaimBondedLP(msg.sender,asset);
+            if(claimA>0){
+               BondVault(bondVault).cFMember(asset,msg.sender);
+            }
+        return true;
+    }
```

Comment:
Add helper 'check' function to see if a member has a bond claim available on an asset (to calling easier in the UI)